src/database/schema/ChannelListSchema.ts

```diff
@@ -106,7 +109,7 @@ class ChannelList implements BaseDbSchema {
 
     try {
       jsonString = JSON.stringify(this.rawJson);
-      db.executeSql(
+      await db.executeSql(
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `await` keyword has been added to the `db.executeSql` function call. This is a good practice as it ensures that the promise returned by the function is resolved before moving on to the next line of code, preventing potential race conditions.
```diff
-      db.executeSql(
+      await db.executeSql(
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->